CMakeLists.txt

```diff
@@ -566,14 +565,39 @@ if (WITH_CLI)
     add_subdirectory (src/appleseed.cli)
 endif ()
 
-if (WITH_STUDIO)
-    add_subdirectory (src/appleseed.studio)
-endif ()
-
 if (WITH_PYTHON)
+    find_package (PythonLibs REQUIRED)
+
+    # Splitting version string into list.
+    string (REGEX MATCHALL "[0123456789]+" PYTHON_VERSION_LIST "${PYTHONLIBS_VERSION_STRING}")
+
+    if (PYTHON_VERSION_LIST)
+        # Accessing major and minor version numbers with the version list we got.
+        list (GET PYTHON_VERSION_LIST 0 PYTHON_MAJOR_VERSION)
+        list (GET PYTHON_VERSION_LIST 1 PYTHON_MINOR_VERSION)
+    else ()
+        # Default to latest Python 2 release if version string cannot be read.
+        message (STATUS "Cannot find Python version. Using 2.7 by default")
+        set (PYTHON_MAJOR_VERSION "2")
+        set (PYTHON_MINOR_VERSION "7")
+    endif ()
+
+    if (PYTHON_MAJOR_VERSION STREQUAL "3")
+        find_package (Boost 1.55 COMPONENTS python3)
+    else ()
+        find_package (Boost 1.55 COMPONENTS python)
```

Comment:
Sorry, this is my mistake. We need to move this line to appleseed.python and appleseed.studio cmake files.
The problem now is that we add the boost python lib to the list of libs to link everywhere.
As it is now we link libappleseed.so, appleseed.cli, ..., with boost python.